scripts/create_bridget_pr.sh

```diff
@@ -0,0 +1,29 @@
+# Usage: `./update_bridget.sh
```

Comment:
I'm surprised the script works without starting with something like `#!/bin/bash`, I guess it's not mandatory!

I'd consider adding one even if it's not mandatory. A trick is that you can use that line to configure the debug options, like `#!/bin/bash -xe` instead of using `set` later